apps/src/applab/applab.js

```diff
@@ -1332,10 +1335,17 @@ Applab.onCodeModeButton = function() {
 /**
  * If the filename is relative (contains no slashes), then prepend
  * the path to the assets directory for this project to the filename.
+ *
+ * If the filename is external and non-https, route it through the MEDIA_PROXY.
  * @param {string} filename
  * @returns {string}
  */
 Applab.maybeAddAssetPathPrefix = function (filename) {
+
+  if (/^http:\/\//.test(filename)) {
```

Comment:
Using Regexp.new('^http://') will make this easier to read.

Also, making the Regexp as a class constant is more self documenting and slightly faster, probably because it compiles the regexp only once. (I verified this an experiment.)  For example:

HTTP_REGEXP= RegExp.new('^http://')
...
if (HTTP_REGEXP.test(filename)) {